.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] great job adding the node_modules inside your .gitignore file, as part of the requirement you need to list test.MD as well, and since the file has already been pushed to your repo I suggest deleting the test.md file from your project and then list it to your gitignore file